src/commands/mod.rs

```diff
@@ -2201,19 +2196,20 @@ fn resolve_destination_revs(
 
 #[instrument(skip_all)]
 fn cmd_new(ui: &mut Ui, command: &CommandHelper, args: &NewArgs) -> Result<(), CommandError> {
+    if args.allow_large_revsets {
```

Comment:
I may have missed it, but I think you need to adjust the docstring (clap help text) for this argument and perhaps make it hidden.

Reply:
Oh, good catch. I've hidden them and updated the docstring.